.cirrus.yml

```diff
@@ -807,7 +807,54 @@ success_task:
         CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
         TEST_ENVIRON: container
     clone_script: *noop
-    script: /bin/true
+    script: *noop
+
+
+artifacts_task:
+    name: "Artifacts"
+    alias: artifacts
+    only_if: *not_docs
+    depends_on:
+        - success
+    # This task is a secondary/convenience for downstream consumers, don't
+    # block development progress if there is a failure in a PR, only break
+    # when running on branches or tags.
+    allow_failures: $CIRRUS_PR != ''
+    container: *smallcontainer
+    env:
+        CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+        TEST_ENVIRON: container
+        CURL: "curl --fail --location -O https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}"
+    clone_script: *noop
+    script:
+        - cd $CIRRUS_WORKING_DIR
+        # Assume the latest Fedora release build is most useful
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/podman
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/podman-remote
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/rootlessport
+        - chmod +x podman* rootlessport
+        # Architecture in filename & can't use wildcards in a URL
+        - mkdir -p $CIRRUS_WORKING_DIR/alttmp && cd $CIRRUS_WORKING_DIR/alttmp
+        - $CURL/Alt%20Arch.%20Cross/gosrc.zip && unzip gosrc.zip
+        - cd $CIRRUS_WORKING_DIR
+        - mv $CIRRUS_WORKING_DIR/alttmp/*.tar.gz ./
```

Comment:
If you've just cd'ed to `$CIRRUS_WORKING_DIR` (in the line right above), why do you need to repeat `$CIRRUS_WORKING_DIR`? Same in lines 846-847.

Reply:
Oops, I think it's a leftover from iterative development.